README.md

```diff
@@ -0,0 +1,60 @@
+This is the Microverse Todo list project using  html + css + modular javascript, ES6 syntax, and Webpack.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+To Do List
+
+Use webpack to bundle JavaScript.
+Learn how to use proper ES6 syntax.
+Use ES6 modules to write modular JavaScript.
+
+
+## Built With
+- Html
+- css
+- javascript
+- Linter
+
+## Live Demo (if available)
+
+[Live Demo Link]()
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+You can clone this repo by typing `git clone https://github.com/markotuya0/To-do-list` on your terminal.
+## Authors
+
+👤👤 **mark otuya**
+
+- GitHub: [@Markotuya0](https://github.com/markotuya0)
+- Twitter: [@mark__anthonny](https://twitter.com/mark__anthonny)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mark-otuya-6a09a5232/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/markotuya0/To-do-list/issues).
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [OPTIONAL]  I can see you are using the Microverse template here, good job 👏 but I suggest adding `MIT.md` file to your project [MIT.md Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and don't forget to customize it 👌.

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs.